bin/compile

```diff
@@ -22,6 +22,7 @@ NC='\033[0m' # No Color
 CURL="curl -s -L --retry 15 --retry-delay 2" # retry for up to 30 seconds
 
 DefaultGoVersion="go1.6"
+GlideVersion="0.10.2"
```

Comment:
Is this something we could make overridable by the users environment? Like, do we want to dictate the versions?